GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -0,0 +1,242 @@
+﻿using CommandLine;
+using GVFS.Common;
+using GVFS.Common.Database;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.Tracing;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace GVFS.CommandLine
+{
+    [Verb(
+        SparseVerb.SparseVerbName,
+        HelpText = @"List, add, or remove from the list of folders that are included in VFS for Git's projection.
+Folders need to be relative to the repos root directory.")
+    ]
+    public class SparseVerb : GVFSVerb.ForExistingEnlistment

```

Comment:
Looks like the comment was dropped when this class was renamed.  Should we print out a message about this being experimental until we get feedback on its behavior (similar to the messages we print for dehydrate/repair)?

cc: @jrbriggs for his thoughts as well.

Reply:
I'm on the fence for this because will we need to add the `--confirm` to the sparse verb as well then?

I should add some text here that explains when there are zero folders, everything will be projected.

Reply:
I personally don't think we'd need a `--confirm`, just a message to dissuade anyone from taking a hard dependency on the current behavior.

Reply:
I think just prepending "EXPERIMENTAL: " to the help text will suffice for letting users know this entire command is subject to change.